main.go

```diff
@@ -65,5 +65,5 @@ func main() {
 		console.Fatalln("Go runtime version check failed.", err)
 	}
 
-	minio.Main(os.Args, os.Exit)
+	minio.Main(os.Args)
```

Comment:
You can't test using recover() only panic() can be tested and panic() doesn't happen inside CLI niether does fatalIf calls panic()